### PR TITLE
add cls target

### PR DESCRIPTION
**Pull request by hhou435**

### CODE REVIEW

**Comment by zhezhaoa** [comment]
Thanks for your work.
Could your please provide commands of using ClsTarget for pre-training and describe corpus format requirements?

**Comment by hhou435** [comment]
The formats of the corpus for cls target are as follows：

```
label1 text1
label2 text2
label3 text3
```

or：

```
label1 texta1 textb1
label2 texta2 textb2
label3 texta3 textb3
```

Label and instance are separated by \t .The label ID should be an integer between (and including) 0 and n-1 for n-way classification. 

We firstly preprocess the book review corpus. We need to specify the model's target in pre-processing stage (*--target cls*):

```
python3 preprocess.py --corpus_path corpora/book_review_cls.txt --vocab_path models/google_zh_vocab.txt --dataset_path dataset.pt \
                      --processes_num 8 --target cls
```

After pre-processing, the raw text is converted to *dataset.pt*, which is the input of *pretrain.py*. We must input the number of prediction labels(*--labels_num*) . Suppose we have a machine with 8 GPUs.:

```
python3 pretrain.py --dataset_path dataset.pt --vocab_path models/google_zh_vocab.txt --pretrained_model_path models/google_zh_model.bin \
                    --output_model_path models/book_review_model.bin  --world_size 8 --gpu_ranks 0 1 2 3 4 5 6 7 \
                    --total_steps 5000 --save_checkpoint_steps 1000 \
                    --embedding word_pos_seg --encoder transformer --mask fully_visible \
                    --target cls --labels_num 2
```
